main.tf

```diff
@@ -0,0 +1,73 @@
+module "label" {
+  source      = "git::https://github.com/cloudposse/terraform-null-label.git?ref=tags/0.5.3"
+  namespace   = "${var.namespace}"
+  name        = "${var.name}"
+  stage       = "${var.stage}"
+  environment = "${var.environment}"
+  delimiter   = "${var.delimiter}"
+  attributes  = "${var.attributes}"
+  tags        = "${var.tags}"
+
+  additional_tag_map = {
+    "propagate_at_launch" = "true"
+  }
+}
+
+resource "aws_launch_configuration" "default" {
+  count = "${var.launch_configuration_enabled == "true" ? 1: 0}"
+
+  name_prefix                 = "${format("%s%s", module.label.id, var.delimiter)}"
+  image_id                    = "${var.image_id}"
+  instance_type               = "${var.instance_type}"
+  iam_instance_profile        = "${var.iam_instance_profile}"
+  key_name                    = "${var.key_name}"
+  security_groups             = ["${var.security_groups}"]
+  associate_public_ip_address = "${var.associate_public_ip_address}"
+  user_data_base64            = "${var.user_data_base64}"
```

Comment:
Just a comment, I would recommend including this in the example.

Reply:
thanks @MoonMoon1919 
I'll update it